### PR TITLE
feat(layouts): lazyload images with lazysizes

### LINKED ISSUE #30: Fix page load time
Currently the Hugo Theme Site has a 60s page load time as measured by WebPageTest:

screen shot 2017-04-25 at 8 01 31 pm

While page load may not be the important metric it once was, it is still important. Here's why:

1. Theme site currently contains 165 images.
1. The 165 images represent 33966.8KB of image data.
1. Every new user visiting the theme site home page using a visual browser will attempt to download all 165 of those images.
1. Downloading 33966.8KB of images concurrently causes jank in the UI and hurts users on data-capped internet plans.
1. Browser busy indicators (the spinners) will typically spin for 60 seconds while the page loads images, even tough the user may only need or want to view a small portion of them.

screen shot 2017-04-25 at 8 08 40 pm

PR to follow to fix this issue. But I wanted to get #29 in first because as it stands the analytics for this site are not being measured effectively.

### PR DESCRIPTION
**Pull request by ghost**
dramatically reduces page load time, jank and image data consumption

fixes #30, relates to https://github.com/spf13/hugoThemes/issues/244

### CODE REVIEW

**Comment by bep** [comment]
@digitalcraftsman might have more insight, but we already have a lazy loader:

https://github.com/spf13/hugoThemesSite/commit/b0a25ab9157e7c7aea9a4fe2dae83be2db45f1f3

Not saying which one is better, we cannot have two!

**Comment by ghost** [comment]
@bep It does not work. I will remove unveil.

**Comment by bep** [comment]
It works fine for me in Chrome.

**Comment by ghost** [comment]
> It works fine for me in Chrome.

It doesn't work for me in Chrome, nor does it work for WebPageTest. Please see the metrics I've provided. I've removed the previous loader, which is subpar because:

1. It uses jQuery and has to wait for the entire page to parse, and jQuery and all other jQuery dependencies to load, as configured, before activating
2. It should not be unsetting the `src` attribute of images. Lazysizes [properly uses those](https://github.com/aFarkas/lazysizes#lqipblurry-image-placeholderblur-up-image-technique) for LQIP, and whenever a `src` attribute is provided a browser _will_ try and load them if the script isn't fired immediately (which may explain why it _works on your machine_).

**Comment by jhabdas** [comment]
The old loader has been removed. The new one auto-initializes. Please test before release. I expect your page load time in WebPageTest will drop approximately 90%.

File: layouts/partials/header.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
     {{ range .Site.Authors }}
       {{ with .twitter }}<meta name="twitter:creator" content="@{{ . }}"/>{{ end }}
     {{ end }}{{ end }}
-
+    <script async defer src="/js/lazysizes.min.js"></script>
```
Comment (ghost) [inline layouts/partials/header.html:35]: Note `defer` here is not necessary but has been added as a convenience for older versions of Internet Explorer, which do not make use of the `async` attribute.

**Comment by bep** [comment]
OK; let us test it.

**Comment by ghost** [comment]
Thanks for the merge. Here are the updated metrics:
http://speed.habd.as/themes_gohugo_io-cable/?period=day

screen shot 2017-04-26 at 7 25 27 am

**Pull request merged**